File: newuserui/src/app/Model/Item.ts
export class Item {
    itemId:number;
    itemName:string;
    itemMaterial:string;
    itemBrandName:string;
    itemSize:string;
    itemPrice:number;
    itemStock:number;
    itemImage1:string;
    itemImage2:string;
    itemImage3:string;
    itemType:string;
    itemAdded:any;
    itemColor:string
    itemCategory:string;

    constructor(id:number,name:string,material:string,Bname:string,price:number,stock:number,size:string,type:string,img1:string,img2:string,img3:string,date:any,cat:string){
        this.itemBrandName = Bname;
        this.itemId =id;
        this.itemImage1 = img1;
        this.itemImage2 =img2;
        this.itemImage3 = img3;
        this.itemMaterial = material;
        this.itemPrice = price;
        this.itemSize = size;
        this.itemStock = stock;
        this.itemType = type;
        this.itemName = name;
        this.itemAdded = date;
        this.itemCategory = cat;
    }
}

File: BackEnd/jeanstation/JeanStationWhishlistService/Models/Item.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace JeanStation.WishlistService.Models
{
    public class Item
    {
        public int ItemId { get; set; }
        public string ItemName { get; set; }
        public string ItemDescription { get; set; }
        public string ItemMaterial { get; set; }
        public string ItemCategory { get; set; }
        public DateTime DateAdded { get; set; }
        public int ItemPrice { get; set; }
        public string ItemColor { get; set; }
        public string ItemBrandName { get; set; }
        public string ItemType{ get; set; }
        public string ItemSize { get; set; }
        public int ItemStock { get; set; }
        public string ItemImage1 { get; set; }
        public string ItemImage2 { get; set; }
        public string ItemImage3 { get; set; }
        

    }
}
